public/app/features/alerting/unified/components/silences/SilencesTable.tsx

```diff
@@ -172,7 +181,14 @@ function SilenceList({
         cols={columns}
         isExpandable
         dataTestId={dataTestId}
-        renderExpandedContent={({ data }) => <SilenceDetails silence={data} />}
+        renderExpandedContent={({ data }) => {
+          return (
+            <>
+              <Divider />
+              <SilenceDetails silence={data} />
```

Comment:
It felt strange not having this separate slightly. With other changes, this is the before...

![image](https://github.com/grafana/grafana/assets/4377076/e649f6a0-32ea-4b88-8f50-4a2e6c3d481f)


and after

![image](https://github.com/grafana/grafana/assets/4377076/b834386b-30e1-4761-8319-95b34f5f24c4)

Reply:
Much nicer!